make/test.mk

```diff
@@ -26,7 +26,7 @@ test-with-coverage: generate
 	@echo "running the tests with coverage..."
 	@-mkdir -p $(COV_DIR)
 	@-rm $(COV_DIR)/coverage.txt
-	$(Q)go test -vet off ${V_FLAG} $(shell go list ./... | grep -v /test/e2e) -coverprofile=$(COV_DIR)/coverage.txt -covermode=atomic ./...
+	$(Q)go test -vet off ${V_FLAG} $(shell go list ./... | grep -v /cmd/manager) -coverprofile=$(COV_DIR)/coverage.txt -covermode=atomic ./...
```

Comment:
doesn't that mean that it won't run the tests inside `cmd/manager` directory? We have one small test there already: https://github.com/codeready-toolchain/host-operator/blob/master/cmd/manager/main_test.go and I'm not sure if skipping it is a good idea

Reply:
No, no. It doesn't affect the tests itself. Only the coverage reporting. I've tested before creating a PR :)

Reply:
Ok :+1:

Reply:
good! I had the same concern as @MatousJobanek ;)